src/components/DateTimePicker/DatePicker/Generate/generateRangePicker.tsx

```diff
@@ -230,7 +230,7 @@ export default function generateRangePicker<DateType>(
               clearIconAriaLabelText={clearIconAriaLabelText}
               id={id}
               separator={
-                <span aria-label="to" className={styles.pickerSeparator}>
+                <span role="text" aria-label="to" className={styles.pickerSeparator}>
```

Comment:
why is this role="text"

Reply:
As  discussed with indumati , the role =presentation reported an issue in the ARC tool whereas role=text did not hence indumati asked me to add the same .

Reply:
This is a non-existant role

Reply:
As suggested by indumati added aria-hidden="true" to the same